taxi/views.py

```diff
@@ -1,3 +1,19 @@
+from lib2to3.fixes.fix_input import context
```

Comment:
The module 'lib2to3.fixes.fix_input' is imported but never used. It's a good practice to remove unused imports to keep the code cleaner and more readable.